Dockerfile_binary-linux-amd64

```diff
@@ -0,0 +1,31 @@
+### 1. Docker build stage: Create MkDocs binary
+FROM squidfunk/mkdocs-material:8.1.3 as source
+
+# Install PyInstaller and required tools
+RUN apk add binutils upx
+RUN pip install --upgrade pip # As time of writing, installed pip v21.0.1 does not work, upgraded v21.3.1 does
+RUN pip install pyinstaller
+
+# Create MkDocs binary inluding material theme
+WORKDIR /tmp
+RUN pyinstaller --recursive-copy-metadata mkdocs --collect-all mkdocs \
+  --copy-metadata mkdocs-material --collect-all material --name mkdocs-material \
+  $(pip show mkdocs | grep Location | sed -e 's/^Location: //')/mkdocs/__main__.py
+
+
+### Final Docker image: Serving MkDocs binary
+FROM alpine
```

Comment:
I would suggest to use a tagged version here. Dependabot will do the upgrade. That way we can guaranty the reproducibility of our builds.